hcl/hcl_stack_test.go

```diff
@@ -160,7 +181,8 @@ func TestHCLParserStack(t *testing.T) {
 			},
 		},
 		{
-			name: "empty name",
+			name:      "empty name",
```

Comment:
```suggestion
			name: "empty name",
```

I think the code is not formatted no ? :thinking: